### ISSUE TITLE
Cleanliness of Code

### ISSUE THREAD

**Issue by apreist14** [luckydesignlady/web-iv-portfolio-site#7]
Proficient

-Your code looks good for the most part. It is clear and contains no inline styles or special characters. The only thing I see as a minor issue is that from the way I understood is that if you have a large-number and medium number column grid is that you don't have to put the large-number in your code. I could be wrong but that is how I understood it.

**Comment by e-m-durham** [comment]
Yep, @apreist14 is right, no need to define larger screen sizes the same as smaller screen sizes. Foundation is mobile first so it inherits up. For example, line 25 just needs class="medium-4 columns" not class="large-4 medium-4 columns"
